mono/mini/driver.c

```diff
@@ -350,6 +351,92 @@ domain_dump_native_code (MonoDomain *domain) {
 }
 #endif
 
+static gboolean
+method_should_be_regression_tested (MonoMethod *method, gboolean interp)
+{
+	ERROR_DECL (error);
+
+	if (strncmp (method->name, "test_", 5) != 0)
+		return FALSE;
+
+	if (interp) {
+		static gboolean filter_method_init = FALSE;
+		static const char *filter_method = NULL;
+
+		if (!filter_method_init) {
+			filter_method = g_getenv ("INTERP_FILTER_METHOD");
```

Comment:
(can be done later) we should rename it to `REGRESSION_FILTER_METHOD`. it's not really interpreter specific

Reply:
notes! I'll address that as I add support for running individual tests with wasm.